src/site/content/en/handbook/markup-code/index.md

```diff
@@ -100,9 +100,9 @@ Indent using two spaces.
 Always use straight quotes. (Readers often copy code directly from the site, and smart quotes can break code.)
 
 {% Compare 'worse' %}
-`const imagemin = require(‘imagemin’);`
+<code>const imagemin = require(‘imagemin’);</code>
```

Comment:
I'm surprised you needed to remove the backticks in this scenario. It looks like Compare adds a newline so the markdown parser should kick in (https://github.com/GoogleChrome/web.dev/blob/c36a2b0f605b7a8ba8dc72dd7385701ab2069bd9/src/site/_includes/components/Compare.js#L53). Was it not doing that?

Reply:
It was, but it also directly violates our rule that we do not allow smart quotes in code samples in .md files...